web/src/main/resources/application.properties

```diff
@@ -2,10 +2,6 @@
 spring.jackson.date-format=com.microsoft.azure.simpletodo.configuration.RFC3339DateFormat
 spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
 
-spring.datasource.url=${AZURE_POSTGRESQL_URL}
```

Comment:
If we remove this, the sample can't run in the local, should we mention it in README?

Reply:
How about changing README to this style:

> 1. This sample project is used to teach you how to launch spring-web app into Azure Spring Apps. Refer to [Launch your first web app](.) to get more detailed information. 
> 2. If you want to run this sample in localhost, please add `spring.datasource.url=` in `application.properties` manually. This property can be configured automatically in Azure Spring Apps, but it cannot be configured automatically in localhost.

I'll add this content in README after the link ([Launch your first web app](.)) is ready.

Reply:
I'm not following this, what does this mean?

but I think we should make this sample runnable on local using a PG docker

and just in case you don't know, when using Azure Service Connector to connect Azure Spring Apps and Azure PostgreSQL, it will inject the following environment to the spring application runtime, so there is no need to have such properties in applciation.properties , see [this doc](https://learn.microsoft.com/en-us/azure/service-connector/how-to-integrate-postgres#java---spring-boot-jdbc) for more detail



* `spring.datatsource.url`
* `spring.datatsource.username`
* `spring.datatsource.password`

This is not using passwordless, please check with service connector team and see what properties they inject into the runtime when using passwordless

I'd expect this one will be there `spring.datasource.azure.passwordless-enabled=true`

Reply:
I am sure when using passwordless with service connector, will inject 3 properties into environment variable:

- spring.datatsource.url=...
- spring.datatsource.username=...
- spring.datasource.azure.passwordless-enabled=true